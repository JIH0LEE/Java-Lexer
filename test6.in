hi "123141231