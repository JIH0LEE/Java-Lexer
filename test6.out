<INTEGER,0>
<INTEGER,0>
<INTEGER,1>
<SEMICOLON>
<INTEGER,0>
<INTEGER,0>
<INTEGER,10>
<SEMICOLON>
<INTEGER,0>
<INTEGER,0>
<INTEGER,10>
<ID,a0010>
<SEMICOLON>
<INTEGER,0>
<INTEGER,0>
<INTEGER,10>
<OP,->
<INTEGER,10>
<SEMICOLON>
<INTEGER,0>
<INTEGER,0>
<INTEGER,10>
<OP,->
<INTEGER,-10>
<SEMICOLON>
