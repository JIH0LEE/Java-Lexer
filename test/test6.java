class ErrTest(){
    String name=" "
    if(3>2){
        name="jiho"
    }
    
}