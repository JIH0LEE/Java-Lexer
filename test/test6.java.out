<1,CLASS,class>
<1,ID,ErrTest>
<1,LPAREN,(>
<1,RPAREN,)>
<1,LBRACE,{>
<2,VTYPE,String>
<2,ID,name>
<2,ASSIGN,=>
<2,LITERAL," ">
<3,IF,if>
<3,LPAREN,(>
<3,NUM,3>
<3,COMP,>>
<3,NUM,2>
<3,RPAREN,)>
<3,LBRACE,{>
<4,ID,name>
<4,ASSIGN,=>
<4,LITERAL,"jiho">
<5,RBRACE,}>
<7,RBRACE,}>
